opensearch-jobs.yml

```diff
@@ -138,6 +138,64 @@ instance_groups:
 #########################################################
 #2nd deploy group - opensearch_data, opensearch_dashboards, ingestors
 #########################################################
+- name: archiver
+  instances: 1
+  jobs:
+  - name: bpm
+    release: bpm
+  - consumes: *consumes-opensearch-manager
+    name: opensearch
+    properties:
+      opensearch:
+        jvm_options:
+        - -Dlog4j2.formatMsgNoLookups=true
+        migrate_data_path: true
+    release: opensearch
+  - name: archiver_syslog
+    properties:
+      logstash:
+        queue:
+          max_bytes: 30gb
+      logstash_archiver:
+        files: 16384
+      logstash_ingestor:
+        outputs:
+        - options:
+            bucket: logs-opensearch-cf-development
```

Comment:
This should come from a Terraform output like it does here: https://github.com/cloud-gov/cg-deploy-logsearch/blob/main/logsearch-jobs.yml#L341C21-L341C79